stimpacks/deploy/deploy.go

```diff
@@ -48,6 +49,19 @@ func (d *Deploy) Run() {
 
 	// Read in the config file and set up defaults
 	d.parseConfig()
+	if d.config.Global.RequiredVersion != "" {
+		d.log.Info("Deploy has set RequiredVersion to:{}, currently:{}", d.config.Global.RequiredVersion, d.stim.GetVersion())
+		if semver.Compare(d.stim.GetVersion(), d.config.Global.RequiredVersion) != 0 {
+			d.log.Fatal("Stim is not at the Required version for deploy, current:{}, required:{}\n\t Please check https://github.com/PremiereGlobal/stim/releases for new versions", d.stim.GetVersion(), d.config.Global.RequiredVersion)
+		}
+	} else {
```

Comment:
elseif instead?

Reply:
actually should be no else, technically both can be set so we should check both.

Reply:
yeah, either way - although you do bomb out if both are set

Reply:
good point